### PR TITLE
Load csv data

### PR DESCRIPTION
**Pull request by georgeamccarthy**
- Created helpers.py
- Wrote da_from_csv in helpers.py
- Loaded protein data in app.py

Note: it may be possible to use [jina.types.document.generators.from_csv](https://github.com/jina-ai/jina/blob/master/jina/types/document/generators.py) but it is not clear to me how this is possible. Missing documentation on required arguments and lack of examples.

### CODE REVIEW

**Comment by georgeamccarthy** [comment]
I think I need you to accept a collaborator invite before I can add you as a reviewer. @fissoreg

**Comment by fissoreg** [comment]
The documentation for the Jina `from_csv` function is here: https://docs.jina.ai/api/jina.types.document.generators/#jina.types.document.generators.from_csv

It is probably a good idea to try to use it.

An example for usage should be similar to this: https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Document.md#construct-from-json-csv-ndarray-and-files

File: backend/helpers.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
+
+from jina import DocumentArray, Document
+
+'''
+Creates a DocumentArray from a chosen column of a csv file.
+'''
```
Comment (fissoreg) [inline backend/helpers.py:6]: I think the docstring is supposed to be inside the function, as per https://www.python.org/dev/peps/pep-0257/

Comment (georgeamccarthy) [reply backend/helpers.py:6]: Good catch!

**Comment by georgeamccarthy** [comment]
> The documentation for the Jina `from_csv` function is here: https://docs.jina.ai/api/jina.types.document.generators/#jina.types.document.generators.from_csv
> 
> It is probably a good idea to try to use it.
> 
> An example for usage should be similar to this: https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Document.md#construct-from-json-csv-ndarray-and-files

I spent around an hour trying to make this work inc: using the function and looking through the source code. I'll post in the support channel for further advice.

**Comment by georgeamccarthy** [comment]
Figured it out, needed to change the data to include a column title then pass that as a dictionary and map it to the text field.

File: backend/app.py [hunk 2]
```diff
@@ -1 +1,6 @@
-from jina import Flow
+from jina.types.document.generators import from_csv
+from jina import DocumentArray
+
+proteins = DocumentArray(
+    from_csv(open("../data/Train_HHblits_1column.csv"), {"Protein sequences": "text"})
```
Comment (fissoreg) [inline backend/app.py:5]: Based on the documentation, the file path should suffice and probably `open` is unnecessary. Not sure though.

**Comment by fissoreg** [comment]
Awesome! I left a little comment but it's a minor thing, feel free to merge when you want.

File: backend/app.py (hunk 2 above)
Comment (georgeamccarthy) [reply backend/app.py:5]: If I just pass it a string then it creates a document array where each document is a letter in the string for some reason. Whatever you pass in gets passed to DictReader which I believe needs an open file https://docs.python.org/3/library/csv.html

**Pull request merged**